### ISSUE TITLE
Eliminate Consul Adapter code check (lint) failures

### ISSUE THREAD

**Issue by leecalcote** [layer5io/meshery-consul#8]
*Description:*
The Meshery adapter for Consul is failing lint checks when building. Currently, these checks are not passing:

```
Found 5 lint suggestions; failing.
129
##[error]/home/runner/work/meshery-consul/meshery-consul/consul/client.go:23:2: a blank import should be only in a main or test package, or have a comment justifying it
130
##[error]/home/runner/work/meshery-consul/meshery-consul/consul/client.go:29:6: type name will be used as consul.ConsulClient by other packages, and that stutters; consider calling this Client
131
##[error]/home/runner/work/meshery-consul/meshery-consul/consul/consul.go:39:1: exported method ConsulClient.CreateMeshInstance should have comment or be unexported
132
##[error]/home/runner/work/meshery-consul/meshery-consul/consul/consul.go:309:1: comment on exported method ConsulClient.ApplyOperation should be of the form "ApplyOperation ..."
133
##[error]/home/runner/work/meshery-consul/meshery-consul/consul/yaml.go:65:1: exported method YAMLDecoder.Close should have comment or be unexported
134
##[error]Process completed with exit code 123.
```

**Expected Behavior**
All lint checks should pass.

See an example of these failures here - https://github.com/layer5io/meshery-consul/pull/7/checks?check_run_id=218292837

**Comment by leecalcote** [comment]
@jesuslerma, we would love to have you jump in here.  // @girishranganathan 

Here are this adapter's latest lint issues - https://github.com/layer5io/meshery-consul/commit/32678173563189cec6a59d0238ef57776cb9cf40/checks?check_suite_id=283639511#step:4:129

**Comment by leecalcote** [comment]
Fixed by #22

**Comment by leecalcote** [comment]
@gunjanmimo will you take another look at the changes here? The Docker build is failing - https://github.com/layer5io/meshery-consul/runs/279310972#step:5:256

**Comment by gunjanmimo** [comment]
fixes #22
